packages/client-core/src/components/InstanceChat/InstanceChat.skiptest.tsx

```diff
@@ -29,7 +30,7 @@ describe('Instance Chat Component', () => {
   })
 
   it('displays chat message', async () => {
-    accessChatState().channels.channels.set([
+    useHookstate(getMutableState(ChatState)).channels.channels.get({ noproxy: true }).set([
```

Comment:
React hooks should only be used inside React components, can just use getMutableState here